src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:46651/organizations/org3/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:46651/project/issues?id=sample&pullRequest=6948&issues=AWlY94j9I7VaXCdnurwS&open=AWlY94j9I7VaXCdnurwS)

[ISSUE_KEY]: AWlY94j9I7VaXCdnurwS